src/packages/admin-ui-components/src/spacer/component.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+
+type Props = {
```

Comment:
the typing here (`type Props`) is different from the typing in Alert (`interface AlertProps`), we should aim to keep it consistent.

Reply:
Fixed